.github/workflows/ci.yml

```diff
@@ -35,8 +35,10 @@ jobs:
     name: "Linting"
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
```

Comment:
Could you elaborate why we need to specify 3.x here?

Reply:
v4 of this action refuses to guess which `python-version` to use...
* https://github.com/actions/setup-python/releases/tag/v4.0.0